editor/plugins/material_editor_plugin.cpp

```diff
@@ -151,19 +158,38 @@ void MaterialEditor::_on_light_2_switch_pressed() {
 }
 
 void MaterialEditor::_on_sphere_switch_pressed() {
-	box_instance->hide();
 	sphere_instance->show();
-	box_switch->set_pressed(false);
+	box_instance->hide();
+	quad_instance->hide();
```

Comment:
I standardized the order so every reference to these three shapes in the order of "sphere, box, quad", like they appear in the inspector.